core/server/services/bulk-email/index.js

```diff
@@ -96,6 +71,10 @@ module.exports = {
                 'message-id': messageId
             };
 
+            if (!mailgunInstance) {
```

Comment:
I don't think we need this check as it happens on line 29 :thinking:

Reply:
🤔 The check on line 29 is for `send` fn while `getStats` can be called separately so this will make sure we have the instance before making a call with it?